includes/class-wcmp-export.php

```diff
@@ -858,7 +861,26 @@ public function get_customs_declaration($order ) {
         // Country (=shop base)
         $country = WC()->countries->get_base_country();
 
-        $items = array();
+        $items = $this->get_item_data($order, $default_hs_code, $country);
+        // Select first 5 arrays when you have more than 5 items
+        if (count($items) > self::MAX_ITEMS) {
+            $items = array_slice($items, 0, 5);
+        }
+
+        // Get the total weight of the package
+        $weight = (int) round($this->get_parcel_weight($order) * 1000);
+
+        return compact('weight', 'invoice', 'contents', 'items');
+    }
+
+    /**
+     * @param $order
+     * @param $default_hs_code
+     * @param $country
+     *
+     * @return array
+     */
+    public function get_item_data($order, $default_hs_code, $country){
```

Comment:
UItlijning in deze functie kan iets beter gebruik `command + \`

Reply:
done